web-app/src/components/CreateUser/index.jsx

```diff
@@ -1,126 +1,79 @@
 import React from 'react';
 import { PropTypes as PT } from 'prop-types';
-import { Container } from './styled';
 import container from './container';
-import { Card, Button, Errorbox } from '../common';
-import DatePicker from 'react-datepicker';
+import { Card, Input, Button, Errorbox } from '../common';
+import { Form, ButtonWrap } from './styled';
 
 export const CreateUser = props => {
   const {
-    formData,
+    formElementsArray,
+    formIsValid,
     submitForm,
     formChanged,
-    startDateChanged,
     error,
     success,
     loading,
   } = props;
-  const {
-    forename,
-    surname,
-    email,
-    password,
-    country,
-    employeeRole,
-    startDate,
-  } = formData;
+
+  let form;
+  if (loading) {
```

Comment:
what happens if this isn't set to null? Could you not do if (!loading) then not have to set it to null?